server/.eslintrc.json

```diff
@@ -27,6 +27,7 @@
       "error",
       "never"
     ],
-    "no-restricted-syntax": ["warn"]
+    "no-restricted-syntax": ["warn"],
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/test/**/*.es6"]}]
```

Comment:
Tests should be able to use packages in `devDependencies` without linting errors.